index.html

```diff
@@ -26,12 +26,53 @@ <h1>Hi_</h1>
         molestiae mollitia ad! Doloremque reiciendis, corrupti ducimus minima sapiente quam excepturi,
         velit at odio amet nemo fuga dolorum magni aperiam, praesentium ipsam officia! Nesciunt
         nulla ullam voluptate ipsa nobis qui asperiores ad maiores sint, blanditiis fuga expedita
-         provident eligendi voluptatem pariatur culpa saepe architecto voluptatum ut. Molestias
-         repellendus commodi, consectetur est tempore animi accusantium reiciendis unde.</p>
+        provident eligendi voluptatem pariatur culpa saepe architecto voluptatum ut. Molestias
+        repellendus commodi, consectetur est tempore animi accusantium reiciendis unde.</p>
       <a href="#">DOWNLOAD MY CV</a>
     </section>
-  <!---PRESENTATION--->
+    <!--PRESENTATION-->
+    <!--Education and Employment-->
+    <section id="resume">
+      <article>
+        <h2>Education</h2>
+        <div>
+          <h3>IFS</h3>
+          <h5><date>15/07/2015</date></h5>
+          <p>Lorem ipsum, sit amet consectetur adipisicing elit. Rem ea architecto sed ipsam nihil dolor soluta provident libero impedit reprehenderit, aliquam officiis mollitia nobis quae?</p>
+        </div>
+        <div>
+          <h3>McMaster</h3>
+          <h5><date>01/01/2023</date></h5>
+          <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Rem alias, quia dolorum, nisi hic ad impedit iure natus accusantium tempore perferendis magni soluta debitis vel?</p>
+        </div>
+        <div>
+          <h3>Institute</h3>
+          <h5><date>01/01/2023</date></h5>
+          <p>A quick review about it</p>
+        </div>
+      </article>
+      <article>
+        <h2>Employment</h2>
+        <div>
+          <h3>Army</h3>
+          <h5><date>01/01/2023</date></h5>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique omnis repellendus, quis dolores laudantium blanditiis quod sapiente, velit nihil delectus veniam? Doloribus laboriosam reprehenderit fugit.</p>
+        </div>
+        <div>
+          <h3>MC Engineering</h3>
+          <h5><date>01/01/2019</date></h5>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Recusandae iste, dolore voluptas deleniti culpa hic, neque, facilis labore illum harum exercitationem suscipit reiciendis ea error!</p>
+        </div>
+        <div>
+          <h3>Institute</h3>
+          <h5><date>01/01/2023</date></h5>
+          <p>A quick review about it</p>
+        </div>
+      </article>
+    </section>
+    <!--Education and Employment-->
   </main>
   <footer></footer>
 </body>
+
```

Comment:
Could you please remove this "empty" line?

Reply:
Already Done!